hoomd/md/pytest/test_aniso_pair.py

```diff
@@ -258,16 +279,14 @@ def to_type_parameter_dicts(types, argument_dict):
         [(0.5, -0.5, -0.5), (0.5, 0.5, -0.5), (0.5, 0.5, 0.5), (-0.5, 0.5, 0.5),
          (-0.5, 0.5, -0.5), (-0.5, -0.5, 0.5), (0.5, -0.5, 0.5),
          (-0.5, -0.5, -0.5)],
-        # ellipsoid
-        []
     ]
 
     # ALJ.get_ordered_vertices only works if coxeter can be imported, so we
-    # check the ModuleNotFoundError and don't add it to the list for the
-    # available tests.
+    # check the RuntimeError and don't add it to the list for the available
+    # tests if coxeter is not available.
```

Comment:
I think this comment is incorrect. If the problem was not being able to import coxeter we _should_ be catching a `ModuleNotFoundError`, and maybe we need to add a second except clause here so that developers can run HOOMD tests without coxeter installed. The current except clause is present because `get_ordered_vertices` can fail if it tries to construct a `coxeter.ConvexPolyhedron` with 0 vertices, right? It looks like that is not an issue anymore anyway because of the way that you have split the params into the `alj_arg_dict0` and `alj_arg_dict1` parsing. Am I misunderstanding that?

Reply:
yeah, good catch. It should be catching a `ModuleNotFoundError` and the comment should reflect that. We should still be able to run the test without coxeter installed once I change the `except RuntimeError` to `Except ModuleNotFoundError`.

Reply:
Please test locally in an env without coxeter. The CI test environments all have coxeter installed.

Reply:
The code that attempts to use coxeter uses this code:
```
        try:
            import coxeter
        except ImportError as error:
            raise RuntimeError(
                "Method requires coxeter as a dependency.") from error
```
There is no guarantee that the reason Python cannot import coxeter is because of a `ModuleNotFoundError`. From HOOMD's perspective, the failure to import coxeter is a `FuntimeError` and the `from error` correctly passes the source exception back to the user so they can debug.

This test is working fine without coxeter installed.

If you are looking to improve this code, you could simplify the code by removing the need for coxeter in the tests. Hard-code the faces, areas, and other parameters that are currently pulled from coxeter in `test_alj.py` and `test_aniso_pair.py`. Then all these tests can run whether the user has coxeter installed or not.

The only test that should require coxeter is a unit test for `get_ordered_vertices` that ensures it returns the expected values.